<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmTcpServerClass</class>
 <widget class="QWidget" name="FrmTcpServerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>FrmTcpServer</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTextEdit" name="textEditMain"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="comboBoxData">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_5">
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="checkBoxHexReceive">
        <property name="text">
         <string>16进制接收</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxHexSend">
        <property name="text">
         <string>16进制发送</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxAscii">
        <property name="text">
         <string>控制字符</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxShow">
        <property name="text">
         <string>暂停显示</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxDebug">
        <property name="text">
         <string>模拟设备</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxAutoSend">
        <property name="text">
         <string>定时发送</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxInterval">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>监听地址</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxListenIP">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>监听端口</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditListenPort">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonListen">
        <property name="text">
         <string>监听</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonSave">
        <property name="text">
         <string>保存</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonClear">
        <property name="text">
         <string>清空</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonRemove">
        <property name="text">
         <string>移除</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelCount">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>共0个客户端</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxSelectAll">
        <property name="text">
         <string>对所有客户端</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonListen</sender>
   <signal>clicked()</signal>
   <receiver>FrmTcpServerClass</receiver>
   <slot>on_listen_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>428</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>598</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSave</sender>
   <signal>clicked()</signal>
   <receiver>FrmTcpServerClass</receiver>
   <slot>on_save_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>412</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>631</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonClear</sender>
   <signal>clicked()</signal>
   <receiver>FrmTcpServerClass</receiver>
   <slot>on_clear_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>673</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonRemove</sender>
   <signal>clicked()</signal>
   <receiver>FrmTcpServerClass</receiver>
   <slot>on_remove_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>699</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>FrmTcpServerClass</receiver>
   <slot>on_send_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>485</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>513</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxHexReceive</sender>
   <signal>stateChanged(int)</signal>
   <receiver>FrmTcpServerClass</receiver>
   <slot>stateChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>-58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxHexSend</sender>
   <signal>stateChanged(int)</signal>
   <receiver>FrmTcpServerClass</receiver>
   <slot>stateChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>600</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxAscii</sender>
   <signal>stateChanged(int)</signal>
   <receiver>FrmTcpServerClass</receiver>
   <slot>stateChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>571</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxDebug</sender>
   <signal>stateChanged(int)</signal>
   <receiver>FrmTcpServerClass</receiver>
   <slot>stateChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>531</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>611</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxAutoSend</sender>
   <signal>stateChanged(int)</signal>
   <receiver>FrmTcpServerClass</receiver>
   <slot>stateChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>594</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxInterval</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>FrmTcpServerClass</receiver>
   <slot>stateChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>403</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>668</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxListenIP</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>FrmTcpServerClass</receiver>
   <slot>stateChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>522</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>654</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditListenPort</sender>
   <signal>textChanged(QString)</signal>
   <receiver>FrmTcpServerClass</receiver>
   <slot>stateChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>521</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>659</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxSelectAll</sender>
   <signal>stateChanged(int)</signal>
   <receiver>FrmTcpServerClass</receiver>
   <slot>stateChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>520</x>
     <y>489</y>
    </hint>
    <hint type="destinationlabel">
     <x>614</x>
     <y>492</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_listen_clicked()</slot>
  <slot>on_clear_clicked()</slot>
  <slot>on_save_clicked()</slot>
  <slot>on_send_clicked()</slot>
  <slot>on_remove_clicked()</slot>
  <slot>stateChange()</slot>
 </slots>
</ui>
